TestFlask.API.Tests/UnitTests/InvocationVariableProcessorTest.cs

```diff
@@ -1,21 +1,21 @@
 ﻿using System;
 using System.Collections;
 using System.Collections.Generic;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Moq;
+using NUnit.Framework;
 using TestFlask.API.Cache;
 using TestFlask.API.InvocationVariable;
 using TestFlask.Data.Repos;
 using TestFlask.Models.Entity;
 
 namespace TestFlask.API.Tests.UnitTests
 {
-    [TestClass]
+    [TestFixture]
     public class InvocationVariableProcessorTest
     {
         string ProjectKey = "testProject";
 
-        [TestMethod]
+        [Test]
         public void Should_replace_mail_address_with_variable()
```

Comment:
Can you update test method names to align with existing test methods? For ex:
 => VariableProcessor_ShouldReplaceMailAddress
existing convention is Subject_ShouldDoOrDoesSomething_[WhenParamSetToSomething] 
"When" part is optional if any setup is necessary